### ISSUE TITLE
[Feature] Support for JSON for terraform 0.12

### ISSUE THREAD

**Issue by tmccombs** [pulumi/pulumi-terraform-bridge#1278]
Attempting to import from terraform configuration that contains json .tf.json files fails with "JSON configuration is not supported". I'd like to be able to import from configuration that has .tf.json files in it.

**Comment by Mejato** [comment]
Yes this capability is definitely needed, i.e., conversion from Terraform CDK to Pulumi.
